src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+# Setting up Printers
+
+Most printers are plug-and-play in Pop!\_OS. Add a network or USB connected printers in `Settings` ➞ `Printers`. Clicking `Add a Printer` will automatically search for any connected printers. Printers can also be added manually by clicking `Additional Printer Settings`.
+
+![Printer Settings Screen](/images/printers/printer-settings.png)
+
+---
+
+## CUPS and Printer Compatibility
+
+Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, print queues, and recognizes connected printers. While CUPS recognizes a wide range of printer models, printer functionality may vary. Check if your printer is compatible using [OpenPrinter.com](https://www.openprinting.org/printers).
+
+## Checking Printer Status in CUPS
+
+Open this web page to look at the CUPS (Common Unix Printing System) configuration and status window:
+
+[localhost:631](http://localhost:631)
+
+The status window will show current print jobs, detected printers, and other information about the printing system. If you would like to share this printer with others on your local network, click on the 'Admin' link, under Server, click on the "Share printers connected to this system" and save the changes. Other computers on your network should than see that printer. When there is a prompt for your username and password, use your user name, and password used to login.
+
+![CUPS Status Window](/images/printers/cups-settings.png)
+
+CUPS allows you to manage printers using Terminal commands. See the [CUPS documentation](https://www.cups.org/doc/admin.html) for more information.
+
+## Adding a Printer
+
+Most printers will be automatically added to the computer. If a printer is not automatically added, you can manually add printers in `Settings` ➞ `Printers`.
+
+1. Press the `Super` key and type the word Printers.
+
+2. Choose the `Printers` application in the search box.
+
+3. In the `Printers Application`, click the `Add a Printer...` button and a box will pop up with different options.
```

Comment:
"`Printers Application`" should probably be "`Printers` application" if you're trying to match the previous list item?

This is also not an application; it's a Settings panel (a page in the Settings application).

> a box will pop up with different options.

"different options" from what? You probably just want to say "various options" or omit the "with ___" phrase altogether.